api/v1alpha1/gatekeeper_types.go

```diff
@@ -206,6 +206,10 @@ type WebhookConfig struct {
 
 	// +optional
 	LogLevel *LogLevelMode `json:"logLevel,omitempty"`
+
+	// Set the --log-denies flag to log all deny, dry run and warn failures. The default value is Enabled
```

Comment:
A wording suggestion, but upon consideration I also think the default should be `Disabled`. This would match other settings like `LogMutations`.

```suggestion
	// LogDenies enables the logging of all deny, dry run, and warn failures. The default value is Disabled.
	// See https://open-policy-agent.github.io/gatekeeper/website/docs/violations/#log-denies
```

Reply:
Agree!